docs/CSS.md

```diff
@@ -0,0 +1,138 @@
+### 1. `float`
+
+* *elements:* all
+* *inherited:* no
+
+#### Values
+
+* `none` (default) -- Is a keyword indicating that the element must not float.
+* `left` -- Is a keyword indicating that the element must float on the left side of its containing block.
+* `right` -- Is a keyword indicating that the element must float on the right side of its containing block.
+
+### 2. `margin`
+
+* *elements:* all
+* *inherited:* no
+
+#### Values
+
+* `length` -- Specifies a fixed width. Negative Values are allowed. See for <length> possible units.
+* `percentage` -- A percentage relative to the width of the containing block. Negative values are allowed.
```

Comment:
> `percentage` -- A percentage **relative to the width of the containing block.**

Highlighted for importance. What the percentage value is a percentage _of_ will trip you up.